crates/amaru/src/ledger/store/rocksdb/mod.rs

```diff
@@ -247,6 +245,12 @@ impl Store for RocksDB {
                 })?;
             }
 
+            let path = self.dir.join(snapshot.to_string());
```

Comment:
Nitpicking but: is this related to `Improvements to telemetry` ?

Reply:
Ah right, stumbled on this while testing multiple `import` in a row,  looked like something simple enough I could sneak in :)